OpenRA.Mods.Common/Traits/Cloak.cs

```diff
@@ -264,6 +264,7 @@ public bool IsVisible(Actor self, Player viewer)
 
 		Color IRadarColorModifier.RadarColorOverride(Actor self, Color color)
 		{
+			// TODO: Should this extend to controlling allied units?
```

Comment:
In my option this is a question beyond this PR. I would remove the comment from this PR.